packages/translations/src/languages/en.ts

```diff
@@ -569,6 +569,7 @@ export const enTranslations = {
     currentlyViewing: 'Currently viewing',
     currentPublishedVersion: 'Current Published Version',
     draft: 'Draft',
+    draftHasPublishedVersion: 'Draft (has published version)',
```

Comment:
I tried `Draft (has published)` for brevity but it didn't seem entirely clear, and the problem worsened in other languages.